frontend/js/store/reducers.js

```diff
@@ -1,11 +1,8 @@
-import { connectRouter } from 'connected-react-router';
-import { combineReducers } from 'redux';
+import { combineReducers } from '@reduxjs/toolkit';
 
 import { restCheckReducer as restCheck } from './rest_check';
 
-export const createRootReducer = (history) => {
-  return combineReducers({
-    router: connectRouter(history),
-    restCheck,
-  });
-};
+
+export const rootReducer = combineReducers({
```

Comment:
Even though we have only one reducer now, I left the `combineReducers` here because any project will have more them one reducer.